### PR TITLE
Rename tests script param

### PR DESCRIPTION
**Pull request by RobertLucian**
`s3-bucket` is misleading that makes you think of a bucket name such as `cortex-examples` and not of an S3 path such as `s3://cortex-examples/output-dir-for-batch/`.

---

checklist:

- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)

### CODE REVIEW

File: test/e2e/tests/aws/test_batch.py [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
-            "--s3-bucket option is required to run batch tests (alternatively set the "
+            "--s3-path option is required to run batch tests (alternatively set the "
             "CORTEX_TEST_BATCH_S3_BUCKET_DIR env var) )"
         )
 
     e2e.tests.test_batch_api(
         client,
         api,
-        test_bucket=s3_bucket,
+        test_bucket=s3_path,
```
Comment (miguelvr) [inline test/e2e/tests/aws/test_batch.py:38]: nit: change the function parameter as well

**Review by miguelvr — APPROVED** [review]

File: test/e2e/tests/conftest.py [hunk 2]
```diff
@@ -44,24 +44,24 @@ def pytest_addoption(parser):
...
     )
 
 
 def pytest_configure(config):
     load_dotenv(".env")
 
-    s3_bucket = os.environ.get("CORTEX_TEST_BATCH_S3_BUCKET_DIR")
-    s3_bucket = config.getoption("--s3-bucket") if not s3_bucket else s3_bucket
+    s3_path = os.environ.get("CORTEX_TEST_BATCH_S3_BUCKET_DIR")
+    s3_path = config.getoption("--s3-path") if not s3_path else s3_path
```
Comment (miguelvr) [inline test/e2e/tests/conftest.py:58]: nit: update the env var as well

File: test/e2e/tests/aws/test_batch.py (hunk 1 above)
Comment (RobertLucian) [reply test/e2e/tests/aws/test_batch.py:38]: Done!

File: test/e2e/tests/conftest.py (hunk 2 above)
Comment (RobertLucian) [reply test/e2e/tests/conftest.py]: Done!

**Comment by miguelvr** [comment]
LGTM

**Pull request merged**